content/2_Bronze/Ad_Hoc.mdx

```diff
@@ -322,3 +329,29 @@ Of course, ad hoc problems can be
 presented below are generally on the harder side.
 
 <Problems problems="general" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is most useful when solving ad hoc problems?
+    <Quiz.Answer>
+      Being able to quickly recall complex algorithms and data structures.
+      <Quiz.Explanation>
+        Incorrect. Ad hoc solutions may use specific algorithms and data structures, but a majority of the solution relies on observations and understanding the problem.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Exploring lots of cases and making observations.
+      <Quiz.Explanation>
+        Correct. Trying small cases allows you to better understand the problem and helps you make more observations. Certain observations may be a crucial part of the solution.
```

Comment:
i feel like this isn't really unique to ad hoc problems

and also I guess it is a little subjective (altho I agree with your answer here)

Reply:
I see what you mean. I can make it more clear and include how such specifics lead to a better understanding of the problem (which is unique to ad hoc problems).